plugin/trino-hive-hadoop2/conf/hive-tests-defaults.sh

```diff
@@ -9,4 +9,3 @@ then
 fi
 
 export DOCKER_IMAGES_VERSION=${DOCKER_IMAGES_VERSION:-$DEFAULT_DOCKER_VERSION}
-export HADOOP_BASE_IMAGE="${HADOOP_BASE_IMAGE:-ghcr.io/trinodb/testing/hdp2.6-hive}"
```

Comment:
defaults should still declare the HADOOP_BASE_IMAGE

Reply:
Why? We always run with a specific version. The concept of "default" seems confusing -- explicit is better.

Reply:
The concept of "default" has it's usage -- you want to allow simple invocation of scrips from command line, for developer's consumption, without having to specify the config (both in hive tests and in product tests)
otherwise devs need to source config files, whicih is error-prone in a (long running) interactive shell